.circleci/config.yml

```diff
@@ -95,6 +95,11 @@ pytorch_linux_build_defaults: &pytorch_linux_build_defaults
           NAMED_FLAG="export USE_NAMEDTENSOR=1"
         fi
 
+        # dispatch aten ops statically for mobile
+        if [[ ${BUILD_ENVIRONMENT} == *"android"* ]]; then
+          NAMED_FLAG="export USE_STATIC_DISPATCH=1"
```

Comment:
Yeah I think what we do for this depends on what we decide to do for specifying the subset of ops. Can we move forward with a boolean flag for now and change it later if we need to? We can change the name if you prefer, or maybe just pass an existing flag. But as far as I know, there's not a single existing flag that does what we want, because we don't want this to be triggered for internal mobile builds. @ljk53 any thoughts?

Reply:
I lean towards having separate flags for now. We might still need decide how to specify ops whitelist, e.g. what if we want to choose between ops having same name? what if we want to use config file instead of encoded string? We can always make the "static dispatch" flag "intern_" flag and set it automatically later (IMO keeping it as separate intern flag is easier to understand anyway).

BTW, you probably only need modify .jenkins/pytorch/build.sh (see how DBUILD_CAFFE2_MOBILE is set there).

Reply:
Is the plan for PyTorch Mobile to like, completely ignore OpenGL, Metal, Vulkan, etc?

Reply:
@ajtulloch when we get there can we use switch-case or hashmap-of-function-pointers approach? The first step is to get rid of huge aten vtable which blocks linker from striping out unused code...

Reply:
Static dispatch doesn't imply only CPU - the code below already generates the switch statements (which is good). We should add a configurable filter on dispatch ids (basically selecting which devices to compile), but it can be done separately.